Hw6_1.py

```diff
@@ -0,0 +1,36 @@
+class Person(object):
+    def __init__(self, name, age, money):
+        self.name = name
+        self.age = age
+        self.money = money
+        print("Information about the person:", name, age, money)
+
+
+class Dealer(Person):
+    def start_game(self):
```

Comment:
it seems here should be as a parameters gamers who will participate in game. 
here also should be generated game results.